src/components/CareerEvent/CareerEvent.js

```diff
@@ -1,5 +1,30 @@
 import React from "react";
 
-export default function CareerEvent() {
-  return <div className="careerEvent"></div>;
-}
+//STYLE dependencies
```

Comment:
Do we still need the CareerEvent file since this was supposed to essentially just become a part of CareerResource?

Reply:
Deleted